src/main/flight/pid.c

```diff
@@ -134,7 +134,9 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .itermLimit = 150,
         .throttle_boost = 0,
         .throttle_boost_cutoff = 15,
-        .iterm_rotation = false                 
+        .iterm_rotation = false,
+        .vbatThrottleCompensationEnabled = 0,
+        .vbatThrottleCompensationVoltage = 35 // Target voltage in 0.1V steps
```

Comment:
I'm not sure this is a good default value.  3.5v is the lower limit that you'd normally fly your batteries so if a pilot enables the feature and doesn't change the default they'll get a significant throttle reduction in the current implementation or no boost in the requested change to not perform reductions.  Maybe a default of 3.8v might be better.  This would give a give some improvement later in the battery life but not lead to crazy gains when initially turned on.